addons/metrics-server/README.md

```diff
@@ -0,0 +1,20 @@
+# Kubernetes Metrics Server
+
+Compatibility matrix:
+
+Metrics Server | Metrics API group/version | Supported Kubernetes version
+---------------|---------------------------|-----------------------------
+0.2.x          | `metrics.k8s.io/v1beta1`  | 1.8+
+0.1.x          | `metrics/v1alpha1`        | 1.7
+
+
+In order to deploy metrics-server in your cluster run the following command from
+the top-level directory of this repository:
+
```

Comment:
@justinsb @chrislovecnm any concern with using directories for addon management vs single manifests?

Reply:
@robinpercy Oh, I see. Actually, I think single file would be best so that people aren't forced to clone the repo or somehow copy the folder. One file would be easier as you can `kubectl apply` the raw github file.

Reply:
@robinpercy @chrislovecnm changed to single file in https://github.com/kubernetes/kops/pull/4581/commits/ae032eb0cc4cfde53fca95d7e50b757cc6ab99aa.